discovery-infra/utils.py

```diff
@@ -541,6 +541,12 @@ def get_network_leases(network_name):
 def create_ip_address_list(node_count, starting_ip_addr):
     return [str(ipaddress.ip_address(starting_ip_addr) + i) for i in range(node_count)]
 
+def create_ip_address_nested_list(node_count, starting_ip_addr):
```

Comment:
lets move all changes to discovery-infra/test_infra/utils.py
I want to deprecate this one

Reply:
Got it. I was wondering why there were two :)

Reply:
BTW they are already there - an exact copy. So when you decide to get rid of this one just change the import (it currently seems to be used in some places, not related to this PR).